include/feature_lib.h

```diff
@@ -0,0 +1,49 @@
+#ifndef FEATUREDETECT_FEATURELIB_H_
+#define FEATUREDETECT_FEATURELIB_H_
+
+#include <vector>
+#include <iostream>
+#include <cstdint>
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/features2d.hpp>
+#include <opencv2/imgproc.hpp>
+
+namespace libstructs {
+
+struct KeyPoint {
+  float x;
+  float y;
+  float size;
+  float angle;
+  float response;
+};
+
+struct CalcOutputStruct {
+  std::vector<KeyPoint> keypoints;
+  std::vector<uint8_t> descriptors;
+};
+
+class FeatureDetector {
+ public:
```

Comment:
You should provide setters for `width` and `height` so that a user won't have to reinitialize the whole class to change them

Reply:
done